src/pages/consequences.module.scss

```diff
@@ -0,0 +1,56 @@
+@import '../assets/shared-styles/colors.module.scss';
+
+.riskBuying {
+    display: inline-block;
+    font-family: cera-light;
+    font-size:180%;
+    width:300px;
+}
+
+.offenderRegistry {
+    display: inline-block;
+    font-family: cera-bold;
+    font-size: 20px;
+    width: 300px;
+}
+
+.getArrested {
+    display: inline-block;
+    font-family: cera-bold;
+    font-size:20px;
```

Comment:
1. Needs a space between the property and the value
2. Font sizes should be in REM units for responsiveness

Reply:
Added a space and chaged from px to rem units